proc-macro/src/store.rs

```diff
@@ -89,21 +85,27 @@ pub fn store(s: Structure) -> TokenStream {
         .filter_map(|bi| bi.ast().attrs.iter().filter_map(parse_returns_attr).next())
         .next()
         .expect("#[store(returns = ..)] needs to be specified.");
+    let filtered_fields: Vec<_> = fields
+        .iter()
+        .filter(|(field, _)| field.to_string() != "_runtime")
```

Comment:
Could we ignore marker fields by `Phantom<T>` rather than by the field name which is brittle?